### PR TITLE
Wifi setup

### PR DESCRIPTION
**Pull request by HeavyKid**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -5,4 +5,6 @@ webserver/exec_file.txt
 password
 webserver/www/exec_file.txt
 *.pyc
+.idea/*
```
Comment (SaschaMzH) [inline .gitignore]: This can be deleted becaus it is dublicated by **.idea**

**Comment by SaschaMzH** [comment]
So, i did not tested this version on a real device. I will do this later.

**Comment by SaschaMzH** [comment]
[Wifi](https://trello.com/c/doTBAZVp/10-wifi)

File: install.sh [hunk 2]
```diff
@@ -16,7 +16,7 @@ opkg update
 # install needed packages
 echo "Install needed packages"
 opkg install curl python-light python-pip pyOnionI2C
-pip install flask
+pip install flask, flask-socketio
```
Comment (JonasTrampe) [inline install.sh]: That must be included in the image builder.

**Comment by SaschaMzH** [comment]
Please document the new API commands in **/webserver/templates/api.html**

**Pull request merged**